shared/js/background/broken-site-report.js

```diff
@@ -58,6 +58,18 @@ export function fire (querystring) {
     load.url(url)
 }
 
+/**
+ * @type {Object<import('../../../packages/privacy-grade/src/classes/trackers').ActionName, string>}
+ */
+const requestCategoryMapping = {
+    ignore: 'ignoreRequests',
+    block: 'blockedTrackers',
+    redirect: 'surroagtes',
```

Comment:
```suggestion
    redirect: 'surrogates',
```

Reply:
Heh just caught that in testing (the only reason I pushed to git)